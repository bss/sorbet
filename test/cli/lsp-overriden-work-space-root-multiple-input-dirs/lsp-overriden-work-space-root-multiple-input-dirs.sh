main/sorbet --silence-dev-message --lsp --disable-watchman --dir test/cli/lsp-overriden-work-space-root-multiple-input-dirs/dir_a test/cli/lsp-overriden-work-space-root-multiple-input-dirs/dir_b --override-lsp-workspace-root test/cli/lsp-overriden-work-space-root-multiple-input-dirs/dir_a 2>&1