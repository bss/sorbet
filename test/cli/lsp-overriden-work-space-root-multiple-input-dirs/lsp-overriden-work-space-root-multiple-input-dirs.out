Sorbet's language server requires a single input directory when used with --override-lsp-workspace-root
